web_programming/instagram_pic.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from bs4 import BeautifulSoup
+import datetime
+
+if __name__ == "__main__":
+    url = input("Enter image url: ")
```

Comment:
```suggestion
    url = input("Enter image url: ").strip()
```
As discussed in CONTRIBUTING.md.